applications/directory/src/components/ProfileReviewComplete.js

```diff
@@ -0,0 +1,49 @@
+import Card from '@mui/material/Card';
+
+const ProfileReviewComplete = () => {
```

Comment:
Let's move this component inside the `ProfileReviewCompletePage` file and delete this file since the page is small and this component is not going to be reused.